doc/nrf/gs_updating.rst

```diff
@@ -123,6 +123,27 @@ Whenever you update to a newer release of the |NCS|, check the :ref:`gs_recommen
       Then, :ref:`set up the build environment in SES <setting_up_SES>` again.
 ..
 
+.. _gs_updating_ses_packages:
+
+Updating SES packages
```

Comment:
@tejlmand , do you think we need this section? I think it might be useful to cover some corner cases, but me and @OeySa would like to know your opinion...